docs/Introduction-QuickStartGuide.md

```diff
@@ -568,6 +568,14 @@ export default {commit};
 
 In the simplest case above, we just need to pass an `optimisticResponse` option, which should refer to an object having the same shape as the mutation response payload. When we pass this option, Relay will know to immediately update our local data with the optimistic response, and then update it with the actual server response or roll it back if an error occurs.
 
+Please note that the actual query and response payload may not have the exact same shape as the selection in your code, because sometimes Relay will select extra fields for you, and you need to add these fields to your optimistic response. For example:
```

Comment:
could you specify "when" Relay selects extra fields for you? i.e. during the compilation step.

I wonder if it's clearer to say, "Relay will add extra fields to your query", instead of "selecting extra fields"

Reply:
I was using "add", but seeing @alloy keeps mentioning "select" makes me think that's a more accurate word 😆.

Will do.

Reply:
Yeah a field entry in a GraphQL document is called a ‘selection’ in the GraphQL nomenclature.